HW_1/task_2.py

```diff
@@ -0,0 +1,14 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+Zuravl = int(input('Введите количество журавликов '))
+
+
+if Zuravl % 6 != 0:
+    print('Задача нерешаема')
+else:
+    Petr = Zuravl / 6
+    Sergey = Zuravl / 6
```

Comment:
Эти строки можно объединить в одну.
```suggestion
    Petr = Sergey = Zuravl // 6
```
Также должен заметить, что в Python не принято использовать в качестве имен переменных транслитерацию кириллических словесных конструкций (русских слов).